### PR TITLE
 Update secure erase to use latest progress notification api

### PR DESCRIPTION
**Pull request by pengz1**
Changes include:
1. Existing secure_erase.py use a progress notification payload like:
{"taskID": xxx, "progress": {"value":0, "maximum"}}
Our latest progress notification payload require payload to be like:
{"taskID": xxx, "value":0, "maximum"}
Update secure_erase.py to adopt latest payload format.
2. Set default secure erase in Progress class, otherwise progress notification may not posted to RackHD
3. Other changes include: 
    Change Progress class to new type;
    Correct Message to message;
    Set default pass_count to 1.

### CODE REVIEW

**Comment by pengz1** [comment]
@yyscamper @lanchongyizu

File: data/templates/secure_erase.py [hunk 1]
```diff
@@ -244,12 +242,12 @@ def __run(self):
                     erase_start_flags[index] = os.path.exists(self.path + value + '.log')
             total_percent = sum(percentage_list)/disk_count
             if total_percent == float('+inf'):
-                payload["progress"]["percentage"] = "Not Available"
+                payload["percentage"] = "Not Available"
             else:
-                payload["progress"]["percentage"] = str("%.2f" % total_percent) + "%"
-                payload["progress"]["value"] = int(total_percent)
+                payload["percentage"] = str("%.2f" % total_percent) + "%"
+                payload["value"] = int(total_percent)
```
Comment (yyscamper) [inline data/templates/secure_erase.py]: In latest progress design, the property `percentage` will be ignored, any progress need be split into `value` and `maximum`. In your case, you could set a constant `maximum` as `100`.

Comment (pengz1) [inline data/templates/secure_erase.py]: maximum is 100, percentage was already there, I just sent in case progress can save calculation.

**Comment by pengz1** [comment]
test this please

File: data/templates/secure_erase.py (hunk 1 above)
Comment (yyscamper) [inline data/templates/secure_erase.py]: Thanks, make sense

**Pull request merged**